Git_Problem3.R

```diff
@@ -0,0 +1,57 @@
+library(tidyverse)
```

Comment:
Here you have included the code for problem 2 as well. This is totally fine, but then I would rename the file and delete the file for problem 2.